app/locale/vi.coffee

```diff
@@ -291,69 +291,69 @@ module.exports = nativeDescription: "Tiếng Việt", englishDescription: "Vietn
 #    google_classroom_blurb1: "Yup! Be sure to use the Google Single Sign-On (SSO) Modal to sign up for your teacher account. If you already have an account using your Google email, use the Google SSO modal to log in next time. In the Create Class modal, you will see an option to Link Google Classroom. We only support rostering via Google Classroom at this time."
 #    google_classroom_blurb2: "Note: you must use Google SSO to sign up or log in at least once in order to see the Google Classroom integration option."
 #    how_much_does_it_cost: "How much does it cost to access all of the available courses and resources?"
-#    how_much_does_it_cost_blurb: "We customize solutions for schools and districts and work with you to understand your use case, context, and budget. __contact__ for further details! See also our __funding__ for how to leverage CARES Act funding sources like ESSER and GEER."
+#    how_much_does_it_cost_blurb: "We customize solutions for schools and districts and work with you to understand your use case, context, and budget. __contact__ for further details! See also our __funding__ for how to leverage CARES Act funding sources like ESSER and GEER." #
```

Comment:
remove `#`

Reply:
This is the exact copy from [en.coffee](https://github.com/codecombat/codecombat/blob/master/app/locale/en.coffee#L293) I realized vi.coffee is a bit out of date compared with the English version, hence the difference and some changes. Do you want me to update it anyway?